fhir-core/src/main/java/com/ibm/fhir/core/util/ResourceTypeHelper.java

```diff
@@ -207,6 +207,18 @@ private static Set<String> getResourceTypesFor(FHIRVersionParam fhirVersion) {
         }
     }
 
+    /**
+     * Get a list of unique resource type names for all possible FHIRVersionParam values
+     * @return
+     */
+    public static Set<String> getAllResourceTypeNames() {
```

Comment:
To me, it would be simpler to use `ResourceType.values()` instead of of getResourceTypesFor each version.

Reply:
mentioned that this is to return concrete types